### PR TITLE
copy edits from peter

### PR DESCRIPTION
**Pull request by peterwgnd**
Copy edits and a few other minor changes in here.

### CODE REVIEW

File: src/css/utils/variables.styl [hunk 1]
```diff
@@ -11,27 +11,10 @@ $mq_vars = {
...
+$sans = Polaris, helvetica, sans-serif;
+$serif = Copernicus, georgia, serif;
 $mono = Menlo, Consolas, Monaco, 'Courier New', monospace
 
-// XXX: quick hack
-$sans = $sans-fb
-$serif = $serif-fb
-$canela = $serif-hed-fb
-$atlas = $sans-fb
-$publico = $serif-fb
```
Comment (colinmorris) [inline src/css/utils/variables.styl]: Not a big deal, but without these aliases, I get some errors in the console about trying to load these fonts.

```load-font.js?ea86:15 A {family: "Canela Web", style: "normal", weight: 300, stretch: "normal"}
handleError @ load-font.js?ea86:15
Promise.catch (async)
loadFontGroup @ load-font.js?ea86:41
(anonymous) @ critical.js?e5e6:29
(anonymous) @ critical.js:71
__webpack_require__ @ critical.js:20
(anonymous) @ critical.js:63
(anonymous) @ critical.js:66
load-font.js?ea86:15 A {family: "Publico Text Web", style: "normal", weight: 400, stretch: "normal"}
handleError @ load-font.js?ea86:15
Promise.catch (async)
loadFontGroup @ load-font.js?ea86:41
(anonymous) @ critical.js?e5e6:30
(anonymous) @ critical.js:71
__webpack_require__ @ critical.js:20
(anonymous) @ critical.js:63
(anonymous) @ critical.js:66
load-font.js?ea86:15 A {family: "Atlas Grotesk Web", style: "normal", weight: 400, stretch: "normal"}
```

I'll look into it.

File: src/html/partials/story/intro.hbs [hunk 2]
```diff
@@ -19,28 +25,28 @@ Popular music is often looked on with cynicism as insipid and endlessly recycled
 
 <div class="prose scrolltext" data-clearsong="1">
 <p class="step unpad-bottom-3">
-The authors analyzed the songs in their dataset using <i>sonic features</i> - that is, properties calculated from the raw audio of the songs.
+	The authors analyzed the sonic features&#8202;—&#8202;—properties like beat, “accousticness,” and “valence,” or mood&#8202;—&8202;for every song that has appeared on the charts since its debut in 1955. The features are calculated directly from the raw audio of the songs.
 </p>
 
 <p class="step unpad-bottom-2" data-song="Believe" data-hide_avg="1">
-Let's go over some of those features. As an example, I'll be using <i>Believe</i> by Cher, which held the #1 spot on the Billboard Hot 100 for four weeks in 1999.
+	Let's go over some of those features. As an example, I'll be using “Believe” by Cher, which held the #1 spot on the Billboard Hot 100 for four weeks in 1999.
```
Comment (colinmorris) [inline src/html/partials/story/intro.hbs]: So you'd prefer quotation marks for titles rather than italics? I noticed there are still some titles in italics, but I'm happy to go through and change those.

I went with italics because I was concerned quotation marks looked a little "busy", especially in paragraphs with lots of them, but I don't have a strong opinion on this.

File: src/html/partials/story/intro.hbs [hunk 3]
```diff
@@ -93,47 +100,47 @@ Sonically, it feels like a much "bigger" song. There are a lot of instruments in
...
 <p class="step" data-highlight_sonics="instrumentalness">
-It's also notable for being a mostly instrumental track. The only words sung are in the repeated hook "Do the hustle".
+	It's also notable for being a mostly instrumental track. The only words sung in the song are the repeated hook “Do the hustle.”
 </p>
 
-<p
-  class="step" data-song="(Shake, Shake, Shake) Shake Your Booty">
-1976's <i>(Shake, Shake, Shake) Shake Your Booty</i> has a very similar high-danceability, high-energy sonic fingerprint.
+<p class="step" data-song="(Shake, Shake, Shake) Shake Your Booty">
+	1976’s disco hit “(Shake, Shake, Shake) Shake Your ” has a very sonic similar fingerprint, with high danceability and high energy.
```
Comment (colinmorris) [inline src/html/partials/story/intro.hbs]: "sonic similar" -> "similar sonic"

(just noting these little fixes so I can remember to make them after merging)

File: src/html/partials/story/intro.hbs [hunk 4]
```diff
@@ -93,47 +100,47 @@ Sonically, it feels like a much "bigger" song. There are a lot of instruments in
...
 </p>
 </div>
 
 <div class="step" data-song="I Will Survive">
 <p>
-Gloria Gaynor's iconic <i>I Will Survive</i> reached number 1 in early 1979, just before the disco bubble popped.</p>
+	Gloria Gaynor's iconic “I Will Survive” reached number 1 in early 1979, just before the disco bubble popped.</p>
 <p class="step" data-highlight_sonics="valence danceability energy">
-The song features an interesting juxtaposition of low valence and with high energy and danceability. It's a classic "dancing through the tears" track.
+	The song features an interesting juxtaposition of low valence and with high energy and danceability. It's a classic “dancing through the tears” track that stood out from its peers in part by dint of a much sparser production than was typical of disco songs at the time.
```
Comment (colinmorris) [inline src/html/partials/story/intro.hbs]: "and with"

File: src/html/partials/story/intro.hbs [hunk 5]
```diff
@@ -93,47 +100,47 @@ Sonically, it feels like a much "bigger" song. There are a lot of instruments in
...
 </p>
 
 <div class="step" data-contrast="Total Eclipse Of The Heart">
 <p>
-Though not as danceable, Bonnie Tyler's power-ballad <i>Total Eclipse Of The Heart</i>, which went to #1 in 1983, is another example of a low-valence song electrified with a relatively energetic production.
+By 1983, emotional ballads like “I Will Survive,” with their roots in Soul music, had mixed with the ascendent pop rock to produce the power ballads of the 1980s. Though not as danceable as “I Will Survive,” Bonnie Tyler's power-ballad “Total Eclipse Of The Heart,” which went to #1 in 1983, similarly electrified a low-valence song with an energetic production.
```
Comment (colinmorris) [inline src/html/partials/story/intro.hbs]: Love this!

File: src/html/partials/story/intro.hbs [hunk 6]
```diff
@@ -216,47 +225,56 @@ TODO: more examples, quick slideshow. Is this song actually atypical? Why? If no
 
 <div class="step" data-song="Royals">
 <p>
-Lorde burst on the scene in 2013 with her singular debut single, <i>Royals</i>.
```
Comment (colinmorris) [inline src/html/partials/story/intro.hbs]: I'm not going to lie, I kind of liked this dumb pun :(

File: src/html/partials/story/intro.hbs [hunk 7]
```diff
@@ -216,47 +225,56 @@ TODO: more examples, quick slideshow. Is this song actually atypical? Why? If no
...
+	Lorde burst on the scene in 2013 with her debut single, “Royals.”
 </p>
 
 <p>
-<i>Royals</i> has been described as "minimalist pop", and its sonic attributes match that description well.
+	“Royals” has been described as “minimalist pop,” and its sonic attributes, all with the exception of dancability dramatically pared down compared to the previous year’s average, match that description well.
```
Comment (colinmorris) [inline src/html/partials/story/intro.hbs]: I like this extra explanation, but I think it makes the sentence a little unwieldy/hard to parse. Maybe the "pared down" part could be a separate sentence?

File: src/css/utils/variables.styl (hunk 1 above)
Comment (peterwgnd) [reply src/css/utils/variables.styl]: Sorry! Didn't even think about that.

File: src/html/partials/story/intro.hbs (hunk 2 above)
Comment (peterwgnd) [reply src/html/partials/story/intro.hbs]: Yeah, we use the Chicago Manual of Style at the school and some folks here are kind of sticklers about it, so I updated to match CMOS's preferred styling. I'll definitely take another pass later, so definitely feel free to correct any that you encounter, but I can always fix later as well

File: src/html/partials/story/intro.hbs (hunk 7 above)
Comment (peterwgnd) [reply src/html/partials/story/intro.hbs]: Works for me

File: src/html/partials/story/intro.hbs (hunk 6 above)
Comment (peterwgnd) [reply src/html/partials/story/intro.hbs]: Ha, put it back and I'll see if I can get it by. I pulled it only because my brain tripped over it as I was reading and it took me a minute to parse.

**Comment by colinmorris** [comment]
This is great! I love the copy edits. And thanks for the work on the CSS.

**Comment by peterwgnd** [comment]
No worries! I'll definitely take another pass on the styling as well – in particular some of the positioning, but I figured I'd let you get back to it since I'd sat on it for an age.

File: src/html/partials/story/intro.hbs (hunk 2 above)
Comment (colinmorris) [reply src/html/partials/story/intro.hbs]: Ah, I just realized why you probably missed some of the titles.

The copy for the intro section where each of the sonic attributes is explained isn't actually in `intro.hbs`. It's stored in js-land (for no particularly good reason), in the file `src/js/sonic-reveal.js`.

If you were just making your edits based on the html file, you may want to look over that js file.

(I already went ahead and changed the title formatting there, but I was thinking you might have some content edits for that section.)
